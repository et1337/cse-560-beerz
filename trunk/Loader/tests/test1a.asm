Test44   .ORIG
         .EXT    Symbol
Begin    ADD     R0, R1, Symbol ; Should generate an I record
Test     .FILL    Symbol    ; Should generate another I record
         .END