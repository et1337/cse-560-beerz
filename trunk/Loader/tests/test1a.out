HTest46MMMM0001
XR0000Symbol
T00001061
E0000